health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -766,175 +766,176 @@
+ 
+       const tabId = url.tabId || "0"; // Default to '0' if tabId is undefined
+       data.body.PlanConfigurationSearchCriteria.status = dic[String(tabId)];
+       cleanObject(data.body.PlanConfigurationSearchCriteria);
+       return data;
+     },
+     additionalCustomizations: (row, key, column, value, t, searchResult) => {
+ 
+       switch (key) {
+         case "ACTIONS":
+           // TODO : Replace dummy file id with real file id when API is ready
+           const dummyFile = "c22a7676-d5d7-49b6-bcdb-83e9519f58df"
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)